docs/UserGuide.md

```diff
@@ -215,16 +215,25 @@ Shows a list of all appointments in the app.
 Updates the status of a property from Option to Purchase, to Sales and Purchase Agreement to Completion
 
 Formats:
-* `update INDEX u/new AMOUNT`
-* `update INDEX u/[proceed][cancel]`
+* `update INDEX OPTION`
 
 Description:
-* Edits the property at the specified `INDEX`. The index refers to the index number shown in the displayed list. The index **must be a positive integer** 1, 2, 3, …​
-* The `new` keyword can only be used on a property without an existing status
-* `proceed` or `cancel` can only be used on a property with an existing status
-* `proceed` would move the status on to the next one. e.g. Option to Sales Agreement or Sales Agreement to Completion
-* `cancel` would remove the status of the property
-* At least one of the optional fields must be provided.
+* Updates the status of the property at the specified `INDEX`. The index refers to the index number shown in the displayed list. The index **must be a positive integer** 1, 2, 3, …​
+  
+Options:
+* `u/new AMOUNT`
+  
+    The `new` keyword can only be used on a property without an existing status
+  
+
+* `u/proceed`
+  
+    Can only be used on a property with an existing status. `proceed` would move the status on to the next one. e.g. Option to Sales Agreement or Sales Agreement to Completion where Option represents the state when the Option to Purchase has been exercised for the property, Sales Agreement represents the state when the Sales and Purchase Agreement has been issued and Completion represents the state when the sales of the property has completed.
```

Comment:
Would prefer a nested table to explain what option, sales agreement and completion mean here. But ignore me if you don't think its better

Reply:
added in a table!

Reply:
omg im so sorry i meant like a nested list like that

> - ...
> - ...

cuz a table would be kind of out of place right?

Reply:
changed the table to a list